opa/opa.go

```diff
@@ -77,13 +87,22 @@ func (o *Opa) WithConfig(ctx context.Context, config *models.Config) error {
 	)
 }
 
-func (o *Opa) Compile(ctx context.Context) error {
+func (o *Opa) Compile(ctx context.Context, skip []string) error {
 	modules := make(map[string]string)
 	err := fs.WalkDir(regoFs, "rego", func(path string, d fs.DirEntry, err error) error {
 		if d.IsDir() {
 			return err
 		}
 
+		if len(skip) != 0 {
+			if filepath.Dir(path) == filepath.Join("rego", "rules") {
+				filename := strings.TrimSuffix(filepath.Base(path), filepath.Ext(path))
+				if slices.Contains(skip, filename) {
+					return err
```

Comment:
Mmmmh return nil here
```suggestion
					return nil
```